scripts/gulpfiles/docs_tasks.js

```diff
@@ -0,0 +1,129 @@
+const { execSync } = require('child_process');
+const { Extractor } = require('markdown-tables-to-json');
+const fs = require('fs');
+const gulp = require('gulp');
+const header = require('gulp-header');
+const replace = require('gulp-replace');
+
+const docsDir = 'docs';
+
+/**
+ * Run API Extractor to generate the intermediate json file.
+ */
+const generateApiJson = function(done) {
+    execSync('api-extractor run --local', { stdio: 'inherit' });
+    done();
+}
+
+/**
+ * API Extractor output spuriously renames some functions. Undo that.
+ * See https://github.com/microsoft/rushstack/issues/2534
+ */
+const removeRenames = function() {
+    return gulp.src('temp/blockly.api.json')
+        .pipe(replace('_2', ''))
+        .pipe(gulp.dest('temp/'));
+}
```

Comment:
It was my understanding that we wanted to use api-extractor in the future for actually analyzing the API and not just for documentation. If so, we would need it as a command-line tool and to have the `.api.json` file written out.

In any case, I'm not really feeling adventurous, my goal was to get API docs written before the release, and at this point move on to higher priority work rather than making this particular task as perfect as possible. I did look into using the JS API but it's not that well documented, and afaik there isn't a matching one for api-documenter.